Shared/Sources/FocusKeyLogger.cpp

```diff
@@ -16,8 +16,10 @@ void FocusKeyLogger::Run(std::shared_ptr<FocusAuthenticator> &authenticator, std
 
     _messageListener->RegisterMessage("FocusSendDataToBackend", [this](const std::string &msg) {
         //TODO: Ensure is thread safe to clear the vector here
-        spdlog::get("console")->info("Clear events cached");
-        _events.clear();
+        if (msg == "OK") {
```

Comment:
What's that?

Reply:
I think it's expecting a success message from the backend although we shouldn't handle it this way, but rather add a `poll` on the queue or set `linger` to 0